src/init.cpp

```diff
@@ -263,7 +263,7 @@ void Shutdown(NodeContext& node)
                 chainstate->ResetCoinsViews();
             }
         }
-        pblocktree.reset();
+        node.chainman->m_blockman.m_block_tree_db.reset();
```

Comment:
Is there anything between this line and `node.chainman.reset()` that requires `pblocktree` to be gone? If not, perhaps we could simply omit this line.

Reply:
Good point, done